src/plugins/vis_type_timeseries/public/application/components/vis_with_splits.js

```diff
@@ -119,7 +123,13 @@ export function visWithSplits(WrappedComponent) {
       );
     });
 
-    return <div className="tvbSplitVis">{rows}</div>;
+    return (
+      <div
+        className={`tvbSplitVis ${visData[model.id].series.length === 1 ? 'tvbSplitVis__one' : ''}`}
+      >
```

Comment:
Instead of repeating existing code, especially well commented code (in the styles), you can just supply a modifier that overrides the necessary styles.

You can also make this easier to read using `classNames()` and creating a reusable variable at the top. And BEM naming uses the `--` syntax for modifiers.

```suggestion
    const hasOneVis = visData[model.id].series.length === 1;

    return (
      <div
        className={classNames('tvbSplitVis', {'tvbSplitVis--one': hasOneVis})}
      >
```

Reply:
done